backend/v1/classes/AptosDomain.js

```diff
@@ -0,0 +1,44 @@
+module.exports = AptosDomain;
+const axios = require("axios");
+
+function AptosDomain(aptosDomain) {
+  this.set(aptosDomain);
+}
+
+//Only needed for DB interactions
+AptosDomain.prototype.set = function setAptos() {
+  if (typeof aptosDomain !== "undefined") {
+    this.description = aptosDomain.description;
+    this.name = aptosDomain.name;
+    this.url = aptosDomain.url;
+  }
+};
+
+AptosDomain.prototype.getAptos = async function getSid(name) {
```

Comment:
[pebble] make sure to rename the second  async function getSid(name) to async function getAptos(name)